runtime/src/main/java/org/corfudb/runtime/clients/ManagementClient.java

```diff
@@ -131,4 +132,18 @@ public QueryResponse queryRequest(UUID workflowId) {
                 .payloadMsg(req));
         return (QueryResponse) CFUtils.getUninterruptibly(resp).getResponse();
     }
+
+    /**
+     * Return a workflow id for this remove operation request.
+     * @param endpoint the endpoint to remove
+     * @return uuid of the remove workflow
+     * @throws Exception
+     */
+    public CreateWorkflowResponse removeNode(String endpoint) throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)

Reply:
I agree with SonarQube here.. Don't throw exception

Reply:
Done.

Reply:
Done.